src/main/java/worldofzuul/presentation/SolarShopController.java

```diff
@@ -0,0 +1,61 @@
+package worldofzuul.presentation;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.MouseEvent;
+import javafx.stage.Stage;
+import worldofzuul.Game;
+import worldofzuul.Input.Command;
+import worldofzuul.Input.CommandWord;
+import worldofzuul.Rooms.Shops.Shop;
+
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+
+public class SolarShopController {
+    @FXML
+    Label solarPrice1, solarPrice2, solarPrice3;
+    @FXML
+    Button btnHouse;
+    @FXML
+    private URL location;
+
+    @FXML
+    public void initialize() {
+        if (solarPrice1 != null) {
+            System.out.println(solarPrice1.getText() + solarPrice2.getText() + solarPrice3.getText());
+        }
+    }
+
+    public void handleBtnHouse(ActionEvent actionEvent) throws IOException {
+        Parent root = FXMLLoader.load(getClass().getResource(("/worldofzuul.presentation/house.fxml")));
+
+        Stage window = (Stage) btnHouse.getScene().getWindow();
+        window.setScene(new Scene(root, 600, 400));
+    }
+
+    public void handleBuyItem(MouseEvent mouseEvent) throws Exception {
```

Comment:
This method is duplicated multiple times

Reply:
What do you suggest?